apps/web/src/views/PositionManagers/components/AddLiquidity.tsx

```diff
@@ -282,7 +283,7 @@ export const AddLiquidity = memo(function AddLiquidity({
               [
                 allowDepositToken0 ? amountA?.numerator ?? 0n : 0n,
                 allowDepositToken1 ? amountB?.numerator ?? 0n : 0n,
-                '0x',
+                manager.name === 'Teahouse Finance' ? slippage : '0x', // hardcode slippage for teahouse temporary
```

Comment:
hmm shall we use id instead of name even for hardcode?

Reply:
btw can we design in a way without hardcoding?

Reply:
I will use Id